pkg/test/e2e/e2eutil/pravegacluster_util.go

```diff
@@ -885,9 +886,25 @@ func CheckStsExists(t *testing.T, f *framework.Framework, ctx *framework.TestCtx
 	if err != nil {
 		return fmt.Errorf("sts doesnt exist: %v", err)
 	}
+
 	return nil
 }
 
+func CheckConfigMapUpdated(t *testing.T, f *framework.Framework, ctx *framework.TestCtx, pravega *api.PravegaCluster, cmName string, field string) error {
+	cm := &corev1.ConfigMap{}
+	err := f.Client.Get(goctx.TODO(), types.NamespacedName{Namespace: pravega.Namespace, Name: cmName}, cm)
+	if err != nil {
+		return fmt.Errorf("failed to obtain configmap: %v", err)
+	}
+	javaopts := cm.Data["JAVA_OPTS"]
```

Comment:
there can also be a check to see if the configmap is `nil` and to check whether it contains the provided key or not. Also the key being passed can be another argument passed to the method to make it more flexible.

Reply:
Done